services/src/main/java/org/apache/druid/cli/CliMiddleManager.java

```diff
@@ -195,6 +202,18 @@ private void configureIntermediaryData(Binder binder)
             biddy.addBinding("deepstore").to(DeepStorageIntermediaryDataManager.class).in(LazySingleton.class);
           }
 
+          @Provides
+          @LazySingleton
+          @Named(ServiceStatusMonitor.TAGS_BINDING)
+          public Supplier<Map<String, Object>> heartbeatDimensions(WorkerConfig workerConfig, WorkerTaskManager workerTaskManager)
+          {
+            return () -> ImmutableMap.of(
+                "workerVersion", workerConfig.getVersion(),
+                "workerCapacity", workerConfig.getCapacity(),
```

Comment:
Not sure about this one. Should it be a separate metric of its own?

Reply:
👍 This is more of a metric than a dimension. Removed.